### PR TITLE
Fix more menu dark theme

### PR DESCRIPTION
**Pull request by cblackuk**
Whilst working on the super secret project I cannot mention, we noticed that more menu dark theme is not in dark theme. This will fix it.

### CODE REVIEW

File: custom_template/styles/docfx.vendor.css [hunk 1]
```diff
@@ -1,1479 +1,1480 @@
...
-.dropdown-menu-right,.dropdown-menu.pull-right{right:0;left:auto}
-.dropdown-header,.dropdown-menu>li>a{display:block;padding:3px 20px;line-height:1.42857143;white-space:nowrap}
-.btn-group>.btn-group:first-child:not(:last-child)>.btn:last-child,.btn-group>.btn-group:first-child:not(:last-child)>.dropdown-toggle,.btn-group>.btn:first-child:not(:last-child):not(.dropdown-toggle){border-top-right-radius:0;border-bottom-right-radius:0}
-.btn-group>.btn-group:last-child:not(:first-child)>.btn:first-child,.btn-group>.btn:last-child:not(:first-child),.btn-group>.dropdown-toggle:not(:first-child){border-top-left-radius:0;border-bottom-left-radius:0}
-.btn-group-vertical>.btn:not(:first-child):not(:last-child),.btn-group>.btn-group:not(:first-child):not(:last-child)>.btn,.btn-group>.btn:not(:first-child):not(:last-child):not(.dropdown-toggle){border-radius:0}
-.dropdown-menu .divider{height:1px;margin:9px 0;overflow:hidden;background-color:#e5e5e5}
-.dropdown-menu>li>a{clear:both;font-weight:400;color:#333}
```
Comment (cblackuk) [inline custom_template/styles/docfx.vendor.css:832]: Changed this

File: custom_template/styles/docfx.vendor.css [hunk 2]
```diff
@@ -1,1479 +1,1480 @@
...
-.fade.in{opacity:1}
-.collapse{display:none}
-.collapse.in{display:block}
-tr.collapse.in{display:table-row}
-tbody.collapse.in{display:table-row-group}
-.collapsing{height:0;overflow:hidden;-webkit-transition-timing-function:ease;-o-transition-timing-function:ease;transition-timing-function:ease;-webkit-transition-duration:.35s;-o-transition-duration:.35s;transition-duration:.35s;-webkit-transition-property:height,visibility;-o-transition-property:height,visibility;transition-property:height,visibility}
-.caret{display:inline-block;width:0;height:0;margin-left:2px;vertical-align:middle;border-top:4px dashed;border-top:4px solid\9;border-right:4px solid transparent;border-left:4px solid transparent}
-.dropdown-toggle:focus{outline:0}
-.dropdown-menu{position:absolute;top:100%;left:0;z-index:1000;display:none;min-width:160px;padding:5px 0;margin:2px 0 0;font-size:14px;text-align:left;list-style:none;background-color:#fff;background-clip:padding-box;border:1px solid #ccc;border:1px solid rgba(0,0,0,.15);border-radius:4px;-webkit-box-shadow:0 6px 12px rgba(0,0,0,.175);box-shadow:0 6px 12px rgba(0,0,0,.175)}
```
Comment (cblackuk) [inline custom_template/styles/docfx.vendor.css:825]: Changed this by removing background colour - it is now in main.css

**Comment by cblackuk** [comment]
image

**Review by wturner123 — APPROVED** [review]
LGTM

**Review by suehighmoor — APPROVED** [review]
LGTM

**Pull request merged**